app/routes/task_routes.py

```diff
@@ -0,0 +1,127 @@
+from app import db
+from app.models.task import Task
+from datetime import datetime
+from flask import abort, Blueprint, jsonify, make_response, request
+from sqlalchemy import desc, asc
+from .helpers_routes import validate_obj, send_slack_message
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+
+@tasks_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+
+    if "title" not in request_body or \
+            "description" not in request_body:
+        return jsonify({"details": "Invalid data"}), 400
```

Comment:
Good that you're doing validation, but you should probably provide end users information about what data is invalid.